### PR TITLE
Documenting Drag and drop designer forms

### PR DESCRIPTION
**Pull request by SeqLaz**

### CODE REVIEW

File: documentation/how-to/attributes-form.en.md [hunk 1]
```diff
@@ -43,6 +43,12 @@ QField form.
 
 Groups and Tabs from Drag and Drop designer forms are supported.
 
+In the "Attributes Form," you can structure your forms using various containers, such as tabs and groups, and enhance interactivity by incorporating expressions with conditionals. By enabling the drag-and-drop designer, you gain the flexibility to display attribute fields, as well as other widgets that are not directly tied to specific fields (e.g., HTML/QML widgets) and "Relations" of layers.
```
Comment (nirvn) [inline documentation/how-to/attributes-form.en.md:46]: You should also add a link to QGIS' own documentation here: https://docs.qgis.org/latest/en/docs/user_manual/working_with_vector/vector_properties.html#vector-attributes-menu

```suggestion
In the "Attributes Form," you can structure your forms using various containers, such as tabs and groups, and enhance interactivity by incorporating expression-based conditional visibility. By enabling the drag-and-drop designer, you gain the flexibility to re-order attribute fields, as well as gaining access to widgets that are not directly tied to specific fields (e.g., HTML/QML widgets).
```

**Comment by suricactus** [comment]
The QField screenshot looks terrible quality, seems a compression issue?

As we are on it, does it make sense to only show boolean field and the dependent section, instead of the irrelevant fields in the middle? I think it will be more understandable for our users.

**Comment by suricactus** [comment]
Can you restore the previous state when there were two screenshots next to each other, that clearly show what happens when the attribute is false and then is true?

**Comment by suricactus** [comment]
> As we are on it, does it make sense to only show boolean field and the dependent section, instead of the irrelevant fields in the middle? I think it will be more understandable for our users.

Can we also adjust for this? There are too many fields that are just noice for the purposes of the screenshot.

**Review by suricactus — APPROVED** [review]

**Pull request merged**